folium/plugins/realtime.py

```diff
@@ -0,0 +1,90 @@
+from branca.element import MacroElement
+from jinja2 import Template
+
+from folium.elements import JSCSSMixin
+from folium.utilities import JsCode, parse_options
+
+
+class Realtime(JSCSSMixin, MacroElement):
+    """Put realtime data on a Leaflet map: live tracking GPS units,
+    sensor data or just about anything.
+
+    Based on: https://github.com/perliedman/leaflet-realtime
+
+    Parameters
+    ----------
+    start : bool, default True
+        Should automatic updates be enabled when layer is added
+        on the map and stopped when layer is removed from the map
+    interval : int, default 60000
+        Automatic update interval, in milliseconds
+    getFeatureId : function, default returns `feature.properties.id`
```

Comment:
in Folium we use snake_case convention for our arguments, even though the Javascript counter parts use camelCase. Please convert these to snake_case. The `parse_options` function converts these to camelCase. Or, for the functions, you can use the `camelize` function later on.